tools/apiview/parsers/apiview-treestyle-parser-schema/model.tsp

```diff
@@ -0,0 +1,94 @@
+import "@typespec/json-schema";
+
+using TypeSpec.JsonSchema;
+
+@jsonSchema
+
+/** ReviewFile represents entire API review object. This will be processed to render review lines. */
+model CodeFile {
+  PackageName: string;
+  PackageVersion: string;
+  /** version of the APIview language parser used to create token file*/
+  ParserVersion: string;
+  Language: "C"|"C++"|"C#"|"Go"|"Java"|"JavaScript"|"Kotlin"|"Python"|"Swagger"|"Swift"|"TypeSpec";
+  /** Language variant is applicable only for java variants*/
+  LanguageVariant?: "None" | "Spring" | "Android" = "None";
+  CrossLanguagePackageId?: string;
+  ReviewLines: Array<ReviewLine>;
+  /** Add any system generated comments. Each comment is linked to review line ID */
+  Diagnostics?: Array<CodeDiagnostic>;
+}
+
+
+/** ReviewLine object corresponds to each line displayed on API review. If an empty line is required then add a code line object without any token. */
+model ReviewLine {
+  /** lineId is only required if we need to support commenting on a line that contains this token. 
+   *  Usually code line for documentation or just punctuation is not required to have lineId. lineId should be a unique value within 
+   *  the review token file to use it assign to review comments as well as navigation Id within the review page.
```

Comment:
wording here seems strange. is the `lineID` assigned to review comment and navigation ID? or is it use to compare against those?

Reply:
Line ID is not used for comparison. It's only used to identify a line from a given Line ID so comments use it to identify where a comment needs to be listed. Comment stores this Line ID in DB. Similarly, Line ID is used as HREF within the review so a type name can navigate to its type definition.